### ISSUE TITLE
3D wheights for loss function

### ISSUE THREAD

**Issue by carmelrabinov** [keras-team/keras#7692]
Hi, I am new to keras and looked for the answer in the documentation and did not find it:
  
I have a stacked LSTM regression model, my input is in the form of: [n_samples, n_timestamp = 100, n_features = 300] and my out put is [n_samples, n_timestamp = 100, n_features = 4] - meaning vector of 4 features for each timestamp.
I need to use a 3D weighted loss function which is different for each on of them, meaning each sample has a different 2D array of weights in the form of [n_timestamp = 100, n_features = 4].
In addition I need to have the ability to use L1 for the some of the 4 features and L2 for the rest.

from the fit() and compile() documentation I did not understand if it possible. when I pass a 3D weights array to  compile(loss_weights = weights ,sample_weight_mode='temporal') I get en error.

Is there a way to use the compile and fit options for this case?
